{"name": "Login button is disabled", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n            failed while waiting 4 seconds\n            to assert Attribute\n            for first_by('css selector', 'input[formcontrolname=\"userName\"]')\n\n            reason: ConditionMismatchException: condition did not match\n            \texpected: value=\"\"\n            \t  actual: value=\"1\"\n            screenshot: file://C:\\Users\\ssoloshchenko\\.selene\\screenshots\\1536751883199\\screen_1536751883200.png", "trace": "..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\wait.py:13: in wait_for\n    return condition.fn(entity)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\conditions.py:141: in fn\n    return self.match(element.get_actual_webelement())\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\conditions.py:277: in match\n    actual='{name}=\"{value}\"'.format(name=self.name, value=actual))\nE   selene.exceptions.ConditionMismatchException\n\nDuring handling of the above exception, another exception occurred:\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:188: in _wait_with_screenshot\n    return wait_for(entity, condition, timeout, polling)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\wait.py:35: in wait_for\n    reason=reason_string), screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message: \nE               failed while waiting 4 seconds\nE               to assert Attribute\nE               for first_by('css selector', 'input[formcontrolname=\"userName\"]')\nE   \nE               reason: ConditionMismatchException: condition did not match\nE               \texpected: value=\"\"\nE               \t  actual: value=\"1\"\n\nDuring handling of the above exception, another exception occurred:\ntest_login.py:21: in test_try_login\n    LoginPage().user_name.should(be.blank)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:306: in should\n    _wait_with_screenshot(self._webdriver, self, condition, timeout)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:193: in _wait_with_screenshot\n    raise TimeoutException(msg, e.screen, e.stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message: \nE               failed while waiting 4 seconds\nE               to assert Attribute\nE               for first_by('css selector', 'input[formcontrolname=\"userName\"]')\nE   \nE               reason: ConditionMismatchException: condition did not match\nE               \texpected: value=\"\"\nE               \t  actual: value=\"1\"\nE               screenshot: file://C:\\Users\\ssoloshchenko\\.selene\\screenshots\\1536751883199\\screen_1536751883200.png"}, "start": 1536751883800, "stop": 1536751893440, "uuid": "390a0507-227f-4b8d-b996-44c21353882f", "historyId": "4883ff26b07e95221d81024068f524b3", "fullName": "tests.test_login#test_try_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "MOBILE98"}, {"name": "thread", "value": "15696-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}